django-backend/fecfiler/contacts/views.py

```diff
@@ -61,14 +62,39 @@ def committee_lookup(self, request):
         fecfile_committees = list(
             self.get_queryset()
             .filter(Q(committee_id__icontains=q) | Q(name__icontains=q))
-            .order_by("-committee_id")
-            .annotate(result_id=F("committee_id"))
-            .values("name", "result_id")
-            .annotate(id=F("result_id"))[:max_fecfile_results]
+            .values()
+            .order_by("-committee_id")[:max_fecfile_results]
         )
         return_value = {
             "fec_api_committees": fec_api_committees,
             "fecfile_committees": fecfile_committees,
         }
 
         return JsonResponse(return_value)
+
+    @action(detail=False)
+    def individual_lookup(self, request):
+        q = request.GET.get("q")
+        if q is None:
+            return HttpResponseBadRequest()
+
+        max_fecfile_results = 10
+        max_fecfile_results_param = request.GET.get("max_fecfile_results", "")
+        if (
+            max_fecfile_results_param is not None
+            and max_fecfile_results_param.isnumeric()
+        ):
+            max_fecfile_results = int(max_fecfile_results_param)
+
+        fecfile_individuals = list(
+            self.get_queryset()
+            .filter(Q(type__icontains="IND") & (
```

Comment:
I believe this is an exact match with the field value so doesn't need a "contains". Would this work and be more exact?: .filter(Q(type="IND") & ...